include/seastar/core/metrics_api.hh

```diff
@@ -358,7 +363,7 @@ using values_reference = shared_ptr<values_copy>;
 
 foreign_ptr<values_reference> get_values();
 
-shared_ptr<impl> get_local_impl();
+shared_ptr<impl> get_local_impl(int handle = default_handle());
```

Comment:
The handle can be a strong type instead of int

Reply:
Does seastar have any utility for named types? I went looking for one, but didn't find anything.